_pages/how-we-work/software.md

```diff
@@ -40,29 +41,34 @@ All software in use at GSA needs to be approved through the IT Standards process
+The TTS Software Concierge will review the request submitted and will contact you in order to ask further questions regarding your specific needs. This meeting is intended to gather further information, propose possible alternatives, and to determine the best approach to meet your needs.
 
-If the ATO is expired or you cannot find it in GEAR, you can post in the Slack channel [#tts-tech-portfolio](https://gsa-tts.slack.com/messages/tts-tech-portfolio/) to ask if there is an updated ATO. If there is no ATO or the ATO is expired, you will need to go through the ATO process to ensure your software is approved by security. The ATO process can take 4-6 weeks and [costs money](https://docs.google.com/spreadsheets/d/1PokRIGaGl04sxMxEHGwMIRaJDx4OPEXpJ7g69ekDdz8/edit#gid=1451563242z) and there is no guarantee it will be approved. If your need is urgent, try to find something in the [IT Standards]({{site.baseurl}}/software/search/) list that is already approved and meets your needs.
+### Step 3: Analysis
 
-### Step 3: [Submit a micropurchase request.]({{site.baseurl}}/purchase-requests/)
+After the initial meeting has been conducted, the Software Concierge will:
 
-## Other lists and approval processes
+- Reach out to other parts of TTS to find out if others are experiencing the same problem
+- Explore GEAR and the TTS inventory to see what options are already available
+- Look at pricing
+- Determine what kind of ATO would be needed/acquisition would need to happen if the current tools don’t meet the need
 
-Typically, ATOs are for data and systems. They are required for any GSA information system that hosts GSA data. Not everything falls into that category, but they still need approval. Rather than being given the approval to Operate, they are given an Approval to Use (ATU). An ATU is a risk-based approval process for usage of services, features or funcstion on already ATO'd information systems or platforms. Both of these processes are explained in the [ATO & ATU Tech Talk Briefing](https://drive.google.com/file/d/1rjv7eiPSpjv4imX8Pi4ghRyS-aYb48ln/view?usp=sharing).
+Please be patient as the Software Concierge conducts their analysis. You may be asked to participate in a follow-up meeting to discuss any new concerns and/or possible alternatives that may be discovered.
 
-- [Amazon Web Services (AWS) services](https://before-you-ship.18f.gov/infrastructure/aws/#building-systems-that-will-be-deployed-directly-to-aws)
-- [Chrome extensions](https://insite.gsa.gov/topics/information-technology/assistance-and-help-desks/service-catalog/it-service-catalog-google-chrome-extension-request?term=google%20extensions)
-- [GitHub integrations]({{site.baseurl}}/github/#rules)
-- [Google Cloud / G Suite services, including APIs](https://docs.google.com/spreadsheets/d/1h0338doPlHIfslS7Huypzs7TlJTFVw_-98oPnum0Cvo/edit#gid=467863101)
-- [Slack integrations]({{site.baseurl}}/tools/slack/integrations/)
```

Comment:
Can we keep these links on this page, even if it's a section at the bottom?

Reply:
re-incorporated those links at the bottom